core/payment-driver/zksync/src/zksync/mod.rs

```diff
@@ -4,6 +4,6 @@
 
 pub mod wallet;
 
-mod faucet;
+pub mod faucet;
```

Comment:
faucet was private on purpose, it was "exposed" through the `wallet.rs`

Maybe you can use wallet::init_wallet(msg)?

Not 100% sure about this one, since you would need to make the message that is un-related to the purpose of this example

Reply:
I'll try this... leaving for the another commit

Reply:
`wallet::init_wallet` is not an option here, because it creates a wallet using `YagnaEthSigner` instead of `PrivateKeySigner`.

Reply:
Yeah - I just wanted to respond that it's using [wallet::get](https://github.com/golemfactory/yagna/blob/dde6dc99c6150d657544f4bb6467d95998f1285a/core/payment-driver/zksync/src/zksync/wallet.rs#L68) which I shoud stay away from ;)

Reply:
Thanks for trying, lets leave it as is :)